xclim/indices/_agro.py

```diff
@@ -404,11 +405,12 @@ def rolling_drydays_count(
     Returns
     -------
     xarray.DataArray
-      The number of days in periods of minimum {window} days for each resampling period.
+      The total length in days of dry periods of minimum {window} days for each resampling period.
     """
-    thresh = convert_units_to(thresh, pr, "hydro")
+    pram = rate2amount(convert_units_to(pr, "mm/day"))
```

Comment:
Same comment as above.
```suggestion
    pram = rate2amount(pr, "mm")
```